config/default.vcl

```diff
@@ -0,0 +1,22 @@
+# specify the VCL syntax version to use
+vcl 4.1;
+
+# import vmod_dynamic for better backend name resolution
+import dynamic;
+
+# we won't use any static backend, but Varnish still need a default one
+backend default none;
+
+# set up a dynamic director
+# for more info, see https://github.com/nigoroll/libvmod-dynamic/blob/master/src/vmod_dynamic.vcc
+sub vcl_init {
+        new d = dynamic.director(port = "8013");
+}
+
+sub vcl_recv {
+	# force the host header to match the backend (not all backends need it,
+	# but example.com does)
+	set req.http.host = "nginx";
+	# set the backend
+	set req.backend_hint = d.backend("mitxonline.odl.local");
```

Comment:
Is this line how you restrict the cache for only local development?

Reply:
It's kinda restricted by virtue of the fact that it's in the Docker mitxonline-default network - those lines set the target host to "nginx" (which is the name of the web server container and also the DNS name inside the Docker network) and additionally sets the host header correctly so it doesn't get confused.